Modules/Core/Common/src/itkObject.cxx

```diff
@@ -494,6 +494,13 @@ Object::GetCommand(unsigned long tag)
 
 void
 Object::RemoveObserver(unsigned long tag)
+{
+  const auto & thisAsConst = *this;
+  return thisAsConst.RemoveObserver(tag);
```

Comment:
Looks fine to me. You might as well do it by one line of code, using C++17 `std::as_const`:

    return std::as_const(*this).RemoveObserver(tag);

But I see, `Object::AddObserver` also had that local variable, `thisAsConst`.  My fault: commit f2783a9aa0157b50e272545edbcad9c3029195ff  😺 (ITK did not yet use C++17, when I did that commit 🤷) Anyway, no problem.

Reply:
Conversation resolved: this comment of mine is no longer relevant now that you removed the non-const overload. removed